pnpm-workspace.yaml

```
@@ -1,8 +1,10 @@
 packages:
   # all packages in subdirs of packages/ and components/
-  - 'providers/*'
+  - 'providers/**'
```

This change needs to be rolled back too.

```suggestion
  - 'providers/*'
```